packages/gluestick/src/renderer/helpers/getAssetsLoader.js

```diff
@@ -13,22 +13,34 @@ module.exports = function getAssetsLoader(
   loadjsConfig: LoadJSConfig,
   entryPointBundle: string,
   vendorBundle: string,
+  config: Object,
 ): string {
+  const { host, protocol, ports } = config.GSConfig;
+  const hostAddress =
+    process.env.NODE_ENV === 'production'
+      ? ''
+      : `${protocol}://${host}:${ports.server}`;
```

Comment:
Is this reliable? In production we want assets to be served from the CDN location where all of the other assets would be served. Can we not use webpack file loader to url for polyfill.min.js?

Something like:
```
import polyfillUrl from "file-loader!./polyfill.min.js"
```

Reply:
Or better yet, couldn't we use the same strategy as we did here: https://github.com/TrueCar/gluestick/blob/232d039b90dc904c99fde29f5cdda5789d60795a/packages/gluestick/src/renderer/helpers/linkAssets.js#L65 with `getAssetPathForFile`